src/utils/helpers.ts

```diff
@@ -1,4 +1,10 @@
+import { customerKeyMap } from './customers';
+
 // Method that retrieves the value of a (potentially) dotted keyString from a object
 export const getObjectValueByFieldName = (object: any, keyString: string) => {
   return keyString.split('.').reduce((obj, key): any => obj && obj[key], object);
 };
+
+export const getOrganizationIdByOrganizationNumber = (organizationNumber: string): string => {
```

Comment:
er denne generisk nok til å ligge her. Hva med å lage en egen hjelpeklasse for customer elns ?